grails-app/views/comment/_comment.gsp

```diff
@@ -0,0 +1,78 @@
+<div id="commentOutput" class="clearfix">
+    <h3>Comments (<b data-bind="text: total"></b>)</h3>
+    <div id="postComment" class="comment clearfix comment-post">
+        <div class="comment-header">
+        </div>
+
+        <div class="comment-body">
+            <textarea class="boxsizingBorder" data-bind="value: newComment().text" placeholder="Write your comment here."></textarea>
```

Comment:
All input fields should have labels for accessibility (and the label should have a 'for' attribute)